src/System.Private.CoreLib/src/System/Runtime/InteropServices/ComEventsSink.cs

```diff
@@ -55,18 +53,20 @@ public static ComEventsSink RemoveAll(ComEventsSink sinks)
             return null;
         }
 
-        public static ComEventsSink Remove(ComEventsSink sinks, ComEventsSink sink)
+        public static ComEventsSink? Remove(ComEventsSink sinks, ComEventsSink sink)
         {
             Debug.Assert(sinks != null, "removing event sink from empty sinks collection");
             Debug.Assert(sink != null, "specify event sink is null");
 
+            ComEventsSink? toReturn = sinks;
+
             if (sink == sinks)
             {
-                sinks = sinks._next;
+                toReturn = sinks._next;
             }
             else
             {
-                ComEventsSink current = sinks;
+                ComEventsSink? current = sinks;
```

Comment:
should this be `current = toReturn` ?

Reply:
I don't think so, though it wouldn't matter.  Why do you ask?

Reply:
I was just curious if you missed to change this as well.